src/Analyzer.Utilities/SyntaxGeneratorExtensions.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Editing;
+
+namespace Analyzer.Utilities
+{
+    public static class SyntaxGeneratorExtensions
```

Comment:
The problem with this approach is that this will result in loading both the C# and VB implementations into memory and will pull in both C# and VB compiler for a C#-only solution (say). We try to ensure that this doesn't happen. So the way we do the language split is by exporting the implementations through MEF and we'll only instantiate the implementation for a given language.

This is ofcourse moot if you simply use SyntaxGenerator.OperatorDeclaration